nautobot/core/templates/utilities/render_jinja2.html

```diff
@@ -0,0 +1,117 @@
+{% extends "base.html" %}
+{% load form_helpers %}
+{% load helpers %}
+{% load static %}
+
+{% block extra_styles %}
+<style type="text/css">
+    .button-container {
+        margin-bottom: 24px;
+    }
+    #render_jinja_template_form textarea {
+        resize: vertical;
+        white-space: nowrap;
+    }
+
+</style>
+{% endblock extra_styles %}
+
+{% block header %}
+<h1>{% block title %}Jinja Template Renderer{% endblock %}</h1>
+<hr>
+{% endblock %}
+
+
+
+{% block content %}
+<form class="form form-horizontal" onsubmit="handleFormSubmit(event)" id="render_jinja_template_form">
+    <div class="row">
+        <div class="col-lg-6 col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading"><strong>Template</strong></div>
+                <div class="panel-body">
+                    <div class="form-group">
+                        <label class="col-md-3 control-label required" for="id_template_code">Jinja Template</label>
+                        <div class="col-md-9">
+                            <textarea cols="40" rows="10" class="form-control" id="id_template_code" required=""></textarea>
```

Comment:
With hard-coded initial `cols` and `rows` on these textareas, are they still reasonably responsive to different window sizes? Or is there a better way to size them?

Reply:
They are. I actually don't think the cols here do anything because bootstrap sets these to 100% width anyway.